### PR TITLE
Migrate Dockerfile to astronomerinc/ap-base:3.11.5-3

### PR DESCRIPTION
**Pull request by andriisoldatenko**
**Which images are updated by this PR?**:

- [x] curator

**Checklist** (required)

Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.

- [ ] version.txt is updated in the changed image(s)

If a security scan fails for an unchanged image...

- [ ]  a fix has been applied OR...
- [ ]  an [issue]() has been created

If adding a new image:

- [ ] the directory has the same name you intend it to be published as, less a preceding "ap-"
- [ ] the directory includes a file "version.txt", with version matching the underlying software version
- [ ] the directory includes a file "cve-whitelist.yaml", which may be empty
- [ ] the file .github/PULL_REQUEST_TEMPLATE.md is updated to include the image in the checklist
- [ ] execute the script .circleci/generate_circleci_config.py, commit changes to .circleci/config.yml

**Additional notes for your reviewer**:

### CODE REVIEW

**Comment by ryw** [comment]
Thanks @andriisoldatenko - this will resolve https://github.com/astronomer/issues/issues/1191

**Comment by ryw** [comment]
Also @andriisoldatenko could you look at https://github.com/astronomer/issues/issues/448 and https://github.com/astronomer/issues/issues/1190 to see if you can build a path for us to do this in all images, if you agree?

File: curator/Dockerfile [hunk 1]
```diff
@@ -13,7 +13,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM bobrik/curator:5.8.1
+FROM alpine:3.11
```
Comment (andriisoldatenko) [inline curator/Dockerfile:16]: @ryw wdyt to replace to ap-base here too?

Comment (ryw) [reply curator/Dockerfile:16]: yeah let's do it :)

Comment (ryw) [reply curator/Dockerfile:16]: can we switch this to astronomer/ap-base:3.11.5-3?

**Pull request merged**